lib/redlock/client.rb

```diff
@@ -197,16 +197,18 @@ def load_scripts
           Scripts::PTTL_SCRIPT
         ]
 
-        scripts.each do |script|
-          @redis.with { |conn| conn.script(:load, script) }
+        @redis.with do |connnection|
+          scripts.each do |script|
+            connnection.call('SCRIPT', 'LOAD', script)
+          end
         end
       end
 
       def recover_from_script_flush
         retry_on_noscript = true
         begin
           yield
-        rescue Redis::CommandError => e
+        rescue RedisClient::CommandError => e
```

Comment:
seems we probably want to rescue `RedisClient::ConnectionError` here https://github.com/redis-rb/redis-client/issues/92#issuecomment-1433700601

noting https://github.com/redis/redis-rb/blob/v5.0.6/lib/redis/client.rb#L7-L19

Reply:
ah, see https://github.com/leandromoreira/redlock-rb/pull/126